code/datums/spells/summonitem.dm

```diff
@@ -93,7 +93,8 @@
 						var/obj/machinery/portable_atmospherics/P = item_to_retrive.loc
 						P.disconnect()
 						P.update_icon()
-
+					if(istype(item_to_retrive.loc, /obj/structure/disposalholder) || istype(item_to_retrive.loc, /obj/machinery/disposal))//fixes the breaking of disposals. No more bluespace connected disposal bins!
+						break;
```

Comment:
No semicolons needed my dude

Reply:
Was writing C++ code earlier today. Still in my muscle memory it seems